public/config/README_configuration.md

```diff
@@ -29,81 +29,65 @@ The configuration file has four main sections and then several key-value pairs.
 The definition of the widgets placed on the page. The "widgets" property is
 an array of objects uniquely identified with the properties "type" and "id".
 
-#### widget CSS properties
+widget types:
+        socketValue: displayed float value
+        socketButton: emit a value when clicked
+        socketState: display boolean state
```

Comment:
Our user community got used to these widgets being called "Indicator". In addition, I think Indicator may be more intuitive to a high school student.

Reply:
done